hello_helpers/src/hello_helpers/gripper_conversion.py

```diff
@@ -22,7 +23,7 @@ def __init__(self):
         self.open_aperture_m = 0.09 #0.125
         self.closed_aperture_m = 0.0
 
-        self.open_robotis = 70.0
+        self.open_robotis = 50.0
```

Comment:
Was 70 too high? What is the difference between ROS and ROS2 in terms of this value?